event_routing_backends/helpers.py

```diff
@@ -0,0 +1,140 @@
+"""
+Helper utilities for event transformers.
+"""
+import logging
+from datetime import timedelta
+from urllib.parse import parse_qs, urlparse
+
+from dateutil.parser import parse
+from django.contrib.auth import get_user_model
+from isodate import duration_isoformat
+
+# Imported from edx-platform
+# pylint: disable=import-error
+from lms.djangoapps.courseware.courses import get_course_by_id
```

Comment:
This feels like bad coupling and I'm talking to the TNL team to figure out if there may be an alternative to doing this.

Reply:
It looks like the function that uses this only gets used once, why do we need it?